lightning/src/chain/channelmonitor.rs

```diff
@@ -579,13 +579,24 @@ pub enum Balance {
 	/// fees) if the counterparty does not know the preimage for the HTLCs. These are somewhat
 	/// likely to be claimed by our counterparty before we do.
 	MaybeClaimableHTLCAwaitingTimeout {
-		/// The amount available to claim, in satoshis, excluding the on-chain fees which will be
-		/// required to do so.
+		/// The amount potentially available to claim, in satoshis, excluding the on-chain fees
+		/// which will be required to do so.
 		claimable_amount_satoshis: u64,
 		/// The height at which we will be able to claim the balance if our counterparty has not
 		/// done so.
 		claimable_height: u32,
 	},
+	/// HTLCs which we received from our counterparty which are claimable with a preimage which we
+	/// do not currently have. This will only be claimable if we receive the preimage from the node
+	/// to which we forwarded this HTLC before the timeout.
+	MaybePreimageClaimableHTLCAwaitingTimeout {
```

Comment:
Could rename the above to `MaybeTimeoutClaimableHTLC` and rename this to `MaybePreimageClaimableHTLC` for consistency/brevity